ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/transceiver_qsfp_default_channelized.schema.yml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  transceiver_qsfp_default_channelized:
```

Comment:
```suggestion
  transceiver_qsfp_default_mode_channelized:
```

Reply:
updated to `transceiver_qsfp_default_mode_4x10:` to align closer to the EOS CLI

Reply:
moved to transceiver_qsfp_default_mode_4x10_false